xml/vt_qemu_ga.xml

```diff
@@ -0,0 +1,174 @@
+  <title>Running &qemu; GA Commands</title>
+
+  <para>
+   &qemu; GA includes many native commands that do not have their &libvirt;
+   counterparts. Refer to <xref linkend="cha-qemu-ga-moreinfo"/> for the source
+   of their complete list. You can run all of the &qemu; GA commands by using
+   &libvirt;'s general purpose command <command>qemu-agent-command</command>:
+  </para>
+
+<screen>virsh qemu-agent-command <replaceable>DOMAIN_NAME</replaceable> '{"execute":"<replaceable>QEMU_GA_COMMAND</replaceable>"}'</screen>
+
+  <para>
+   For example:
+  </para>
+
+<screen>&prompt.sudo;virsh qemu-agent-command sle15sp2 '{"execute":"guest-info"}' --pretty
+{
+"return": {
+  "version": "4.2.0",
+  "supported_commands": [
+    {
+      "enabled": true,
+      "name": "guest-get-osinfo",
+      "success-response": true
+    },
+[...]
+</screen>
+ </sect1>
+ <sect1 xml:id="cha-qemu-ga-commands">
+  <title>&virsh; Commands that Require &qemu; GA</title>
+
+  <para>
+   Several &virsh; commands require &qemu; GA for their functionality. Here are
+   some of them:
+  </para>
+
+  <variablelist>
+   <varlistentry>
+    <term><command>virsh guestinfo</command></term>
+    <listitem>
+     <para>
+      Prints information about the guest from the guest's point of view.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><command>virsh guestvcpus</command></term>
+    <listitem>
+     <para>
+      Querys or change state of virtual CPUs from guest's point of view.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><command>virsh set-user-password</command></term>
+    <listitem>
+     <para>
+      Sets the password for a user account in the guest.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><command>virsh domfsinfo</command></term>
+    <listitem>
+     <para>
+      Shows a list of mounted file systems within the running domain.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><command>virsh dompmsuspend</command></term>
+    <listitem>
+     <para>
+      Suspends a running guest.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect1>
+ <sect1 xml:id="cha-qemu-ga-libvirt">
+  <title>Enhancing &libvirt; Commands</title>
+
+  <para>
+   If &qemu; GA is enabled inside the guest, a number of &virsh; subcommands
+   have enhanced functionality when run in the <emphasis>agent</emphasis> mode.
+   The following list includes only several examples of them. For a complete
+   list, see the &virsh; manual page and search for the
+   <literal>agent</literal> string.
+  </para>
+
+  <variablelist>
+   <varlistentry>
+    <term><command>virsh shutdown --mode agent</command> and <command>virsh reboot --mode agent</command></term>
+    <listitem>
+     <para>
+      This method of shutdown or reboot is more reliable because the shutdown
+      is performed in a clean state of the guest.
```

Comment:
What does this mean? What is "a clean state of the guest"?

If it just means a clean shutdown, then I would favour:

"... is more reliable, because it leaves the the guest clean for its next run."

Or something like that. But is there any reason that sending it an external shutdown command from the host is cleaner than logging into the guest and running `sudo shutdown` there? I do not see a particular difference.